app/controllers/api/v1/packages_controller.rb

```diff
@@ -199,6 +199,8 @@ def undesignate_stockit_item
 
       def dispatch_stockit_item
         @orders_package = OrdersPackage.find_by(id: params[:package][:order_package_id])
+        return render_order_status_error if @orders_package.order.can_dispatch_item?
```

Comment:
i think the logic is the wrong way round here `can_dispatch_item` suggests the action can proceed but instead we are throwing an error?

also, please call it `package`. We want to remove the usage of the word item as much as possible.